lib/api/include/irods/ticketAPI.hpp

```diff
@@ -0,0 +1,414 @@
+#ifndef IRODS_TICKET_API_HPP
+#define IRODS_TICKET_API_HPP
+
+#include <string_view>
+#include <type_traits>
+#include <iostream>
+#include <string>
+
+#undef NAMESPACE_IMPL
+#undef RxComm
+#undef rxTicketAdmin
+
+#ifdef IRODS_TICKET_ADMINISTRATION_ENABLE_SERVER_SIDE_API
+#    define NAMESPACE_IMPL server
+#    define RxComm         RsComm
+#    define rxTicketAdmin  rsTicketAdmin
+
+struct RsComm;
+#else
+#    define NAMESPACE_IMPL client
+#    define RxComm         RcComm
+#    define rxTicketAdmin  rcTicketAdmin
+
+struct RcComm;
+#endif // IRODS_TICKET_ADMINISTRATION_ENABLE_SERVER_SIDE_API
+
+/**
+ * @brief Namespace to contain the API for tickets
+ *
+ */
+namespace irods::administration::ticket
```

Comment:
Add the `experimental` namespace to this like so:
```cpp
namespace irods::experimental::administration::ticket
```
This gives us room to change the API in the future without raising too many eyebrows.